www/install/createTablesCentstorage.sql

```diff
@@ -157,7 +157,8 @@ CREATE TABLE `log_action` (
   `log_contact_id` int(11) NOT NULL,
   PRIMARY KEY (`action_log_id`),
   KEY `log_contact_id` (`log_contact_id`),
-  KEY `action_log_date` (`action_log_date`)
+  KEY `action_log_date` (`action_log_date`),
+  KEY `action_object_date` (`object_type`,`action_log_date`)
```

Comment:
you should also do an upgrade script to add this key in existing databases
(I don't know the migration time on platforms with lot of logs)

Reply:
I don't want to put it yet. Because we could not cherry-pick in 20.10.x. It should be another PR.

Reply:
You could put it in the upgrade script of 20.10.2, then you will not have any issue on cherry-pick

Reply:
I removed the 20.10.x target in the description, according to this discussion